aepsych/database/tables.py

```diff
@@ -368,11 +371,91 @@ def __repr__(self):
         )
 
     @staticmethod
-    def update(engine):
+    def update(server, engine):
         logger.info("DbRawTable : update called")
 
+        # Get every master table
+        for master_table in server.get_master_records():
+
+            master_id = master_table.unique_id
+
+            tell_messages = (
```

Comment:
I think you can make this more explicit and concise by using `master_record.children_replay` as in https://github.com/facebookresearch/aepsych/blob/main/aepsych/server/server.py#L159

Reply:
On it!